charts/wildfly/templates/ingress.yaml

```diff
@@ -0,0 +1,3 @@
+{{- if (and (not .Values.deploy.route.enabled) .Values.deploy.ingress.enabled) }}
```

Comment:
the route resource should also be updated so that it's not created if the ingress is created.

Reply:
and somewhere making sure that if both are enabled, it's an error :)

Reply:
I was leaving route as the default/ legay behavior